tests/virtualenv/virtualenv.yaml

```diff
@@ -0,0 +1,120 @@
+schemaVersion: "1.0.0"
+commandTests:
+  - name: "env var setup"
+    envVars:
+      - key: "VIRTUAL_ENV"
+        value: "/env"
+      - key: "PATH"
+        value: "/env/bin:$PATH"
+    command: ["echo", "'setup complete'"]
+
+#####################################################
+#    Default Python Tests
+#####################################################
+
+  - name: "python installation"
+    command: ["which", "python"]
+    expectedOutput: ["/usr/bin/python\n"]
+
+  - name: "virtualenv installation"
+    setup: [["virtualenv", "/env"]]
+    command: ["which", "python"]
+    expectedOutput": ["/env/bin/python\n"]
+
+  - name: "python version"
+    command: ["python", "--version"]
+    # we check stderr instead of stdout for Python versions < 3.4
+    # https://bugs.python.org/issue18338
+    expectedError: ["Python 2.7.9\n"]
+
+  - name: "pip installation"
+    command: ["which", "pip"]
+    expectedOutput: ["/env/bin/pip\n"]
+
+  - name: "gunicorn flask"
+    setup: [["pip", "install", "gunicorn", "flask"]]
+    command: ["which", "gunicorn"]
+    expectedOutput: ["/env/bin/gunicorn"]
+      
+  - name: "flask integration"
+    command: ["python", "-c", "\"import sys; import flask; sys.exit(0 if flask.__file__.startswith('/env') else 1)\""]
+
+#####################################################
+#    Python 3.4 Tests
+#####################################################
+
+  - name: "virtual env teardown"
+    command: ["rm", "-rf", "/env"]
+
+  - name: "python installation"
+    command: ["which", "python3.4"]
+    expectedOutput: ["/usr/bin/python3.4\n"]
+
+  - name: "virtualenv python installation"
+    setup: [["virtualenv", "-p", "python3.4", "/env"]]
+    command: ["which", "python"]
+    expectedOutput: ["/env/bin/python\n"]
+
+  - name: "virtualenv python3 installation"
+    command: ["which", "python3"]
+    expectedOutput: ["/env/bin/python3\n"]
+
+  - name: "python version"
+    command: ["python", "--version"]
+    expectedOutput: ["Python 3.4.2\n"]
+
+  - name: "pip installation"
+    command: ["which", "pip"]
+    expectedOutput: ["/env/bin/pip\n"]
+
+  - name: "pip3 installation"
+    command: ["which", "pip3"]
+    expectedOutput: ["/env/bin/pip3\n"]
+
+  - name: "gunicorn flask"
+    setup: [["pip", "install", "gunicorn", "flask"]]
+    command: ["which", "gunicorn"]
+    expectedOutput: ["/env/bin/gunicorn"]
+
+  - name: "flask integration"
+    command: ["python", "-c", "\"import sys; import flask; sys.exit(0 if flask.__file__.startswith('/env') else 1)\""]
+
+#####################################################
+#    Python 3.5 Tests
+#####################################################
+
+  - name: "virtual env teardown"
+    command: ["rm", "-rf", "/env"]
+
+  - name: "virtualenv python installation"
```

Comment:
Add something like lines 49-51 here:

    - name: "python installation"
      command: ["which", "python3.5"]
      expectedOutput: ["/opt/python3.5/bin/python3.5\n"]

And change the virtualenv call to

      setup: [["virtualenv", "-p", "python3.5", "/env"]]